documentation/specs/proposed/RAR_caching_onepager.md

```diff
@@ -0,0 +1,54 @@
+## RAR caching
+
+During every build we need to gather the graph of references and pass
+them to the compiler. This process is highly cacheable as the references
+don’t change all that often. Currently we have some limited caching in
+place however the way nodes are assigned work results in frequent cache
+misses.
+
+### Goals and motivations
+
+1ES team wants to isolate their File I/O related to the which is causing
```

Comment:
I don't understand this sentence, is it missing a part? The scenario needs more clarification. What is I/O isolation in this case?

Reply:
@ccastanedaucf that sounds accurate to me, right?